test/realmodel/src/generateCase.js

```diff
@@ -480,10 +501,11 @@ async function splitContext(context) {
         count ++;
       }
     };
+    let layer = result.length + 1;
     let caseSample = `describe('CTS Real Model Test', function() {
   const assert = chai.assert;
   const nn = navigator.ml.getNeuralNetworkContext();
-  it('Check result for RESHAPE example/${count} of ${JSON_DATA.getModelName()} model', async function() {
+  it('Check result layer-${layer} for RESHAPE example/${count} of ${JSON_DATA.getModelName()} model', async function() {
```

Comment:
@shenx3x Please update case purpose align with above CONV_2D op, using 
_Check result for **layer-${layer}** RESHAPE_
not
_Check result **layer-${layer}** for RESHAPE_